components/dropdown/user/src/index.scss

```diff
@@ -123,7 +123,8 @@
         display: flex;
         position: relative;
 
-        &:hover {
+        &:hover,
+        &Highlight {
```

Comment:
This is OK but I prefer to separate the `selectors`  to the `pseudo-classes`. And to do not duplicate code, you can always usa partials with the `%`.

Something like:
```
// This is not going to be generated as css
%hoverEffect {
    color: color: $c-accent;
}

&:hover {
   @extend %hoverEffect;
   // only hover styles here
}

&:Highlight {
   @extend %hoverEffect;
   // only highlight styles here
}
```

Reply:
Placeholders are fine, i like them. 
They'll output exactly the same code as my original variation but, to be honest, in this case I don't see any improvement because it's is adding just a color property `color: $c-accent;` but you are right, for other cases with more CSS declarations would be useful.

Reply:
Like I said, a totally personal opinion! Well done btw :smile:

Reply:
@tureey, we haven't had good experiences with placeholders here in Schibsted, so somehow they are not allowed 😛. There are "safer" ways to approach it.

Reply:
👍  I am curious though about these experiences hahaha